apis/eks/v1alpha1/identityproviderconfig_references.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The Crossplane Authors.
```

Comment:
small nitpick: change to 2021 in all files

Reply:
Fixed